tickets/src/__tests__/middlewares/web3Middleware.test.js

```diff
@@ -447,21 +445,4 @@ describe('Web3 middleware', () => {
     )
     expect(mockWeb3Service.getKeyByLockForOwner).not.toHaveBeenCalled()
   })
-
```

Comment:
The web3Provider stuff should not be affected by the fact that we do not have a user wallet.